SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveImage.java

```diff
@@ -3,10 +3,12 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.io.Serializable;
+
 /**
  * In-app message background image.
  */
-public class SwrveImage extends SwrveWidget {
+public class SwrveImage extends SwrveWidget implements Serializable {
```

Comment:
I can't see where/why SwrveImage, SwrveCampaignState, SwrveButton need to be Serializable? Can you show me